app/assets/stylesheets/application.scss

```diff
@@ -7,3 +7,51 @@
 @import "base/base";
 
 @import "layout";
+
+div.nav {
+
+	ul.menu {
+		@include outer-container(68em);
+		list-style-type: none;
+
+		li.menu-link {
+			@include span-columns(2);
+
+			font-size: 1.3em;
+			font-weight: bold;
+			margin: 0.5em 0;	
+		}	
+	}
+
+	border-bottom: solid 1px #ddd;
```

Comment:
Color literals like `#ddd` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented with spaces, not tabs